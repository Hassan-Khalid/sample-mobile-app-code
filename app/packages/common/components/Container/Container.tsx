import React from 'react';
import { View } from 'react-native';
import styles from './styles/Container.styles';

const Container = ({ children }) => {
  return (
    <View style={styles.container}>
      {children}
    </View>
  );
}

export default Container;