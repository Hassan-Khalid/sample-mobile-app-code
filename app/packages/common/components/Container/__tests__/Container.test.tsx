import React from 'react';
import { render } from '@testing-library/react-native';
import Container from '../Container';

describe('Container Component', () => {
  beforeEach(() => {
    jest.clearAllMocks();
  });
  
  it('should render container', async () => {
    // exercise
    const wrapper = render(<Container />);
    
    // post-conditions
    expect(wrapper.toJSON()).toMatchSnapshot();
  });
});
