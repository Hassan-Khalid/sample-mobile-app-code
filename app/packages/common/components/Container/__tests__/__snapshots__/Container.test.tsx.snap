// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container Component should render container 1`] = `
<View
  style={
    {
      "borderColor": {
        "dynamic": {
          "dark": "#172B4D",
          "highContrastDark": undefined,
          "highContrastLight": undefined,
          "light": "#172B4D",
        },
      },
      "flexGrow": 1,
    }
  }
/>
`;
